apps/site/assets/ts/stop/components/DepartureTimes.tsx

```diff
@@ -292,18 +297,27 @@ const DepartureTimes = ({
   );
 
   const schedules = schedulesByHeadsign(schedulesForDirection);
-
+  let departures: DepartureInfo[] = [];
+  let id: number = 0;
   return (
     <>
       {Object.entries(schedules).map(([headsign, schs]) => {
-        const formattedTimes = toDisplayTime(
-          schs,
-          predictionsByHeadsign[headsign]
-            ? predictionsByHeadsign[headsign]
-            : [],
-          overrideDate
+        const preds = predictionsByHeadsign[headsign]
+          ? predictionsByHeadsign[headsign]
+          : [];
+        const formattedTimes = toDisplayTime(schs, preds, overrideDate);
+        departures = mergeIntoDepartureInfo(schs, preds);
+        id += 1;
+        return (
+          <div
+            key={`${headsign}-${id}`}
```

Comment:
This could probably just be `${headsign}`.  In this array headsign should be unique.  Also its bad practice to use [the index of a list as a key](https://react.dev/learn/rendering-lists#where-to-get-your-key)

Reply:
Interesting using index was also used in `Headsign.tsx`. `${headsign}` isn't always unique looking at the courthouse stop there are multiple `South Station` headsigns